lib/camp.js

```diff
@@ -351,6 +351,7 @@ function SecureCamp(opts) {
 inherits(SecureCamp, https.Server);
 
 Camp.prototype.handle = SecureCamp.prototype.handle =
+Camp.prototype.use = SecureCamp.prototype.use =
```

Comment:
Is it really useful to be an express lookalike? Especially considering the subtle differences, I find that unnecessary, borderline error-inducing.

Regardless of the decision to add it, it would need to be a separate patch.